tests/scripts/thread-cert/test_netdata_publisher.py

```diff
@@ -285,13 +284,24 @@ def test(self):
             self.assertEqual(len(services), min(num, DESIRED_NUM_DNSSRP_UNCIAST))
             self.verify_unicast_services(services)
 
+        for node in routers:
+            node.srp_server_set_enabled(True)
+            self.simulator.go(WAIT_TIME)
+        self.assertEqual(sum(node.srp_server_get_state() == 'running' for node in routers),
+                         min(len(routers), DESIRED_NUM_DNSSRP_UNCIAST))
+        self.assertEqual(sum(node.srp_server_get_state() == 'stopped' for node in routers),
+                         max(len(routers) - DESIRED_NUM_DNSSRP_UNCIAST, 0))
+
         for node in routers:
             node.netdata_unpublish_dnssrp()
             self.simulator.go(WAIT_TIME)
             num -= 1
             services = leader.get_services()
             self.assertEqual(len(services), min(num, DESIRED_NUM_DNSSRP_UNCIAST))
             self.verify_unicast_services(services)
+        for node in routers:
+            node.srp_server_set_enabled(False)
+            self.assertEqual(node.srp_server_get_state(), 'disabled')
```

Comment:
Here seems a bit strange, though nothing wrong.

may consider putting these before Line 287?

Reply:
Fine to keep it as is.